### DISCUSSION TITLE
Model overrides, well copyright thinks overwise

### DISCUSSION THREAD

**Discussion by likeproblem** [likeproblem/RestockRecolour#13] [Collaborator Disscusion]
@Spartwo https://github.com/PorktoberRevolution/ReStocked/issues/1027#issuecomment-2435799404

So, what do we do?

**Comment by Spartwo** [comment]
My brother in christ, we're potential contributors

**Reply by likeproblem** [reply]
I have read, ChrisAdderley is the creator of both renamed models as of right now, maybe asking for permission to distribute modified models/pull request them will help?

**Comment by likeproblem** [comment]
@Spartwo https://github.com/PorktoberRevolution/ReStocked/issues/1027#issuecomment-2439645016

I have honestly no idea why we can't do that, the mod still requires Restock to be operational, and without it models are practically not applied

Like, we are helping instead right?? If nothing comes out of it, we would be *politely* asked to remove model overrides, and it is no good

**Reply by EmberSkyMedia** [reply]
Ignore those parts, they will remain un-supported in the Recolour. They have a copyright and their wishes must be respected.

**Reply by Spartwo** [reply]
General consensus is that, take the files private. I'll keep the stack seperator .blend and try to get it included. 

Otherwise mark this stuff as known issues with a short reason for why in the readme

**Comment by Spartwo** [comment]
HatBat has written a deliniation handler for TU so meshes can be declared like duplicate modules so we can work around the ReStock staff. It's not out yet but I'll be pushing adjusted  paint configs that are made with it in mind.

**Reply by likeproblem** [reply]
Where can I follow it, sounds great!
